src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/SqlClient/TdsParser.cs

```diff
@@ -3688,7 +3686,7 @@ private bool IsValidAttestationProtocol(SqlConnectionAttestationProtocol attesta
 
 #if ENCLAVE_SIMULATOR
                 case TdsEnums.ENCLAVE_TYPE_SIMULATOR:
```

Comment:
Could we update this constant to follow the same naming?
```suggestion
                case TdsEnums.ENCLAVE_TYPE_NONE:
```

Reply:
This is a constant for an enclave type. There is no new "None" enclave type. The server can still send an enclave type of `SIMULATOR` so it should not be changed.